Cargo.toml

```diff
@@ -3,6 +3,7 @@ members = [
   "crates/client/db",
   "crates/client/exec",
   "crates/client/sync",
+  "crates/client/sync/src/eth",
```

Comment:
I'd like it to be in crates/client/eth directly, this is too much crate nesting

Reply:
fixed